tornado-assembly/src/bin/tornado-test

```diff
@@ -95,6 +95,7 @@ __TEST_THE_WORLD__ = [
     TestEntry("uk.ac.manchester.tornado.unittests.math.TestMath"),
     TestEntry("uk.ac.manchester.tornado.unittests.batches.TestBatches"),
     TestEntry("uk.ac.manchester.tornado.unittests.lambdas.TestLambdas"),
+    TestEntry("uk.ac.manchester.tornado.unittests.functional.TestLambdas"),
```

Comment:
```suggestion
```

Reply:
These unit-tests are already registered in line 54.

Reply:
We have two classes names `TestLambdas`. Let's keep them for this PR and we will unify them later.

Reply:
correct, if you check line 54 it already has the `functional.TestLambdas`

Reply:
Ok, you are right.